README.adoc

```diff
@@ -68,9 +68,37 @@ include::{common-includes}/gitclone.adoc[]
 [role='command']
 include::{common-includes}/twyb-intro.adoc[]
 
-You can find your service at http://localhost:9080/artists.
+// static guide instructions:
+ifndef::cloud-hosted[]
+You can find your service at http://localhost:9080/artists[http://localhost:9080/artists^].
```

Comment:
You can find your service at the http://localhost:9080/artists[http://localhost:9080/artists ^] endpoint.